src/main/java/com/dreammaster/gthandler/recipes/AssemblerRecipes.java

```diff
@@ -97,6 +97,40 @@ public void run() {
         makeElectricMachinePartRecipes();
         makeCircuitPartRecipes();
 
+        // Bronze Hull
+        GTValues.RA.stdBuilder()
+                .itemInputs(
+                        GTOreDictUnificator.get(OrePrefixes.plate, Materials.Bronze, 8),
+                        GTUtility.getIntegratedCircuit(6))
+                .itemOutputs(ItemList.Hull_Bronze.get(1)).duration(2 * SECONDS + 10).eut(TierEU.RECIPE_LV)
+                .addTo(assemblerRecipes);
+
+        // Bricked Bronze Hull
```

Comment:
This conflicts with the Bronze Hull above if you are making a lot of them, since the only thing differentiating them is the addition of Bricks in this Recipe. Having them both on circuit 6 shouldn’t be done.

Reply:
What are the exact rules for a new recipe? Does it need to have at least two different items or a different circuit?

Reply:
If any combinations of items (regardless of their amount) in a recipe can make another recipe, that's a conflict. If there's a conflict the recipe has to be changed to fix it, though this usually means changing the circuit.

Reply:
This still hasn't been fixed.

Reply:
> This still hasn't been fixed.

I send him a message on Discord @Dioxop